From d6e2c2d34d336398f5948a8b731fefff3dc0ff12 Mon Sep 17 00:00:00 2001
From: Christian Hesse <mail@eworm.de>
Date: Thu, 2 Jun 2022 20:49:46 +0200
Subject: [PATCH] shared/utmp-wtmp: fix build without utmp

Commit 16618332388442f2f1c3e52b0a9fde00121564a3 changed a function to
add an extra argument. The data types used when building without utmp
missed the change.

Upstream-Status: Backport [d6e2c2d34d336398f5948a8b731fefff3dc0ff12]
Signed-off-by: Martin Jansa <Martin.Jansa@gmail.com>

---
 src/shared/utmp-wtmp.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/shared/utmp-wtmp.h b/src/shared/utmp-wtmp.h
index 36e4203b4f..188d011fdd 100644
--- a/src/shared/utmp-wtmp.h
+++ b/src/shared/utmp-wtmp.h
@@ -59,7 +59,7 @@ static inline int utmp_wall(
                 const char *message,
                 const char *username,
                 const char *origin_tty,
-                bool (*match_tty)(const char *tty, void *userdata),
+                bool (*match_tty)(const char *tty, bool is_local, void *userdata),
                 void *userdata) {
         return 0;
 }
-- 
2.36.1

